src/schema/v2/searchCriteriaLabel.ts

```diff
@@ -0,0 +1,346 @@
+import { GraphQLObjectType, GraphQLString } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { startCase } from "lodash"
+
+import allAttributionClasses from "lib/attributionClasses"
+import { COLORS } from "lib/colors"
+
+type SearchCriteriaLabel = {
+  /** The GraphQL field name of the filter facet */
+  field: string
+
+  /** The human-friendly name of the filter facet */
+  name: string
+
+  /** The human-friendly value of the filter facet */
+  value: string
+}
+
+/**
+ * A type, derived here in Metaphysics from the upstream Gravity response,
+ * that represents a SearchCriteria's filter in a human-friendly label,
+ * suitable for use in a pill UI, for example.
+ */
+export const SearchCriteriaLabel = new GraphQLObjectType<
+  SearchCriteriaLabel,
+  ResolverContext
+>({
+  name: "SearchCriteriaLabel",
+  description:
+    "Human-friendly representation of a single SearchCriteria filter",
+  fields: {
+    field: {
+      type: GraphQLString,
```

Comment:
Similarly here - unless we anticipate null values, would be best to make the return types non-nullable from the beginning:

```suggestion
      type: new GraphQLNonNull(GraphQLString),
```